caluma/form/migrations/0003_auto_20190128_0836.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.18 on 2019-01-28 08:36
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("form", "0002_auto_20181221_1517")]
+
+    operations = [
+        migrations.CreateModel(
+            name="AnswerDocument",
+            fields=[
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+                ("modified_at", models.DateTimeField(auto_now=True)),
+                (
+                    "created_by_user",
+                    models.CharField(blank=True, max_length=150, null=True),
+                ),
+                (
+                    "created_by_group",
+                    models.CharField(
+                        blank=True, db_index=True, max_length=150, null=True
+                    ),
+                ),
+                (
+                    "id",
+                    models.UUIDField(
+                        default=uuid.uuid4,
+                        editable=False,
+                        primary_key=True,
+                        serialize=False,
+                    ),
+                ),
+                (
+                    "sort",
+                    models.PositiveIntegerField(
+                        db_index=True, default=0, editable=False
+                    ),
+                ),
+            ],
+            options={"ordering": ("-sort", "id")},
+        ),
+        migrations.AddField(
+            model_name="question",
+            name="row_form",
+            field=models.ForeignKey(
+                blank=True,
+                help_text="One row of table is represented by this form",
```

Comment:
Some ideas:
`One row of a table is represented by this form`
`Form that represents a row of the table`
`Form used to enter each row of the table`

Reply:
used second suggestion